Metalama.Migration/src/Metalama.Migration/Aspects/Advices/OnEventInvokeHandlerAdvice.cs

```diff
@@ -4,12 +4,14 @@
 
 using System;
 
+using Metalama.Framework.Advising;
+using Metalama.Framework.Aspects;
+
 namespace PostSharp.Aspects.Advices
 {
     /// <summary>
-    /// There is no equivalent to this feature in Metalama.
+    /// In Metalama, implement the <see cref="IAspect{T}.BuildAspect"/> method and use <c>builder</c>.<see cref="IAspectBuilder.Advice"/>.<see cref="IAdviceFactory.OverrideAccessors(Metalama.Framework.Code.IFieldOrPropertyOrIndexer,in Metalama.Framework.Advising.GetterTemplateSelector,string?,object?,object?)"/>
```

Comment:
This now references the OverrideAccessors method for wrong declaration kind - IFieldOrPropertyOrIndexer.